src/test/java/com/google/sps/data/FakeAvailabilityDaoTest.java

```diff
@@ -0,0 +1,271 @@
+// Copyright 2019 Google LLC
```

Comment:
Discussed with Alex, and we agree that the fake DAOs do not need tests because they are themselves only used in tests. Please read my comments on this file anyway for future reference. I am sorry for all the extra work this caused.